src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/billing-ui.mdx

```diff
@@ -9,7 +9,7 @@ redirects:
   - /docs/accounts/accounts-billing/general-account-settings/introduction-account-settings
 ---
 
-To find your billing- and usage-related UI, go to [one.newrelic.com](https://one.newrelic.com). In the upper right corner, click the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary/#account-dropdown) to see various account settings. You'll see some of these billing-related UI options:
+To find your billing- and usage-related UI, go to [one.newrelic.com](https://one.newrelic.com) and click the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary/#account-dropdown) to see various account settings. You'll see some of these billing-related UI options:
```

Comment:
Screenshot of the dropdown? Is this overkill?

Reply:
i actually removed the screenshot; we can add it back once we only have one, should be soon

Reply:
it's a little touchy because how it appears differs depending on their user model.